plugin/web/extensions/sidebar/modals/datacatalog/components/content/YourDataPage/WebDataTab.tsx

```diff
@@ -62,7 +62,8 @@ const WebDataTab: React.FC<Props> = ({ onOpenDetails, setSelectedWebItem }) => {
         url: dataUrl,
         format: setDataFormat(fileType, filename),
       };
-      if (onOpenDetails) onOpenDetails(item);
+      const requireLayerName = true;
+      if (onOpenDetails) onOpenDetails(item, requireLayerName);
```

Comment:
Shouldn't you check the file type and only set requireLayerName to true if it is a type that needs a layer name (ie WMS)?

Reply:
Yes, I missed this bit.